### PR TITLE
fix: id:45762 add scroller if colum is more in table

### PR DESCRIPTION
**Pull request by Farhan-emgage**
add scroller if colum is more in table

### CODE REVIEW

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/45762

File: src/components/Picker/Picker.tsx [hunk 1]
```diff
@@ -192,6 +191,11 @@ class Picker extends React.PureComponent<Props, State> {
       className = theme.pickerResultShow;
     }
 
+<<<<<<< HEAD
```
Comment (weegrom) [inline src/components/Picker/Picker.tsx:194]: @Farhan-emgage it appears a merge conflict was included in this PR.

File: src/components/Picker/Picker.tsx [hunk 2]
```diff
@@ -408,8 +412,13 @@ class Picker extends React.PureComponent<Props, State> {
       onBlur: autoSuggestMethods.onBlur,
       disabled: readOnly || disabled || (!!this.props.maxSelectedItems && this.props.maxSelectedItems <= chipListState.length),
     };
+<<<<<<< HEAD
```
Comment (weegrom) [inline src/components/Picker/Picker.tsx:415]: @Farhan-emgage it appears a merge conflict was included in this PR.

**Review by weegrom — CHANGES_REQUESTED** [review]

File: src/components/Picker/Picker.tsx (hunk 2 above)
Comment (Farhan-emgage) [reply src/components/Picker/Picker.tsx]: @weegrom The conflict is resolved

**Review by weegrom — APPROVED** [review]

**Pull request merged**